schemas/com.snowplowanalytics.snowplow/identify/jsonschema/1-0-0

```diff
@@ -0,0 +1,25 @@
+{
+        "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+        "self": {
+                "vendor": "com.snowplowanalytics.snowplow",
+                "name": "identify",
+                "version": "1-0-0",
+                "format": "jsonschema"
+        },
+        "description": "A Snowplow identify event.",
+        "type": "object",
+        "properties": {
+                "id": {
+                        "description": "The user identifier.",
+                        "type": ["string", "null"],
+                        "maxLength": 256
+                },
+                "email": {
```

Comment:
I considered suggesting to put a regex on the email field, so that downstream consumers of this entity know it has been validated already.  But email regexes are difficult, and so it might not solve any real-life problem.